guides/doc-Configuring_User_Authentication/master.adoc

```diff
@@ -0,0 +1,67 @@
+include::common/attributes.adoc[]
+include::common/header.adoc[]
+:context: authentication
+
+= {ConfiguringUserAuthenticationDocTitle}
+
+// This guide is not ready for stable releases
+ifdef::HideDocumentOnStable[]
+include::common/modules/snip_guide-not-ready.adoc[]
+endif::[]
+ifndef::HideDocumentOnStable[]
+
+ifdef::satellite[]
+include::common/modules/proc_providing-feedback-on-red-hat-documentation.adoc[leveloffset=+1]
+endif::[]
+
+include::common/assembly_accessing-server.adoc[leveloffset=+1]
+
+include::common/assembly_configuring-an-ldap-server-as-an-external-identity-provider-for-project.adoc[leveloffset=+1]
+
+include::common/assembly_configuring-kerberos-single-sign-on-with-freeipa-in-project.adoc[leveloffset=+1]
+
+// The following ifdef sets :keycloak: to `Red Hat Single Sign-On` for Satellite builds only. For, non-Satellite builds, :keycloak: stays set to `Keycloak`.
+ifdef::satellite[]
+:keycloak: {keycloak-wildfly}
+:keycloak-example-com: rhsso.example.com
+endif::[]
+// The following attribute definitions ensure that modules can be reused between the two included Keycloak assemblies: parent-context saves the original context, context defines a new, unique context to enable the reuse.
+:parent-context: {context}
+:context: keycloak-wildfly
+include::common/assembly_configuring-keycloak-wildfly-authentication-for-project.adoc[leveloffset=+1]
+// The following attribute reloads the original context value from parent-context that was defined earlier.
+:context: {parent-context}
+// The following ifdef unsets :keycloak: because in Satellite builds, it's always either {keycloak-wildfly} or {keycloak-quarkus}.
+ifdef::satellite[]
+:!keycloak:
+endif::[]
+
+ifndef::satellite[]
+// The following ifdef sets :keycloak: to `Red Hat build of Keycloak` for Satellite builds only. For, non-Satellite builds, :keycloak: stays set to `Keycloak`.
+ifdef::satellite[]
```

Comment:
If you set line 39 and exclude this assembly for Satellite, then you won't need to set attribute for Satellite.

built docs for Satellite exclude [Configuring single sign-on and two-factor authentication with Quarkus-based Keycloak in Foreman](https://theforeman-foreman-documentation-preview-pr-3294.surge.sh/nightly/Configuring_User_Authentication/index-katello.html#configuring-single-sign-on-and-two-factor-authentication-with-keycloak-in-project_keycloak-quarkus): https://theforeman-foreman-documentation-preview-pr-3294.surge.sh/nightly/Configuring_User_Authentication/index-satellite.html

I am not blocking your PR but believe it could be even shorter.